packaging/README-dockerhub.md

```diff
@@ -39,3 +41,8 @@ different commit or tag, specify the `SOURCE_COMMIT` build argument:
 ```shell
 sudo docker build --no-cache --build-arg SOURCE_COMMIT=v0.0.4 -t nmstate/centos7-nmstate -f Dockerfile.centos7-nmstate .
 ```
+
```

Comment:
Maybe you should add an OR here, as by mistake someone may think they need to run both commands (the one from line 42).

Reply:
Good point, I will just remove the first line. Not sure if I intended it to be there.